packages/ui/src/domainModules/collectionMammals/components/MammalManager/index.js

```diff
@@ -240,6 +266,10 @@ class MammalManager extends Component {
     )
   }
 
+  handleExportToCsv(event) {
```

Comment:
remove this one, it shouldn't be a main column active tab thing. it will probably be some request to the backend, but it's not implemented yet